modules/apps/data-engine/data-engine-api/src/test/java/com/liferay/data/engine/field/type/util/LocalizedValueUtilTest.java

```diff
@@ -30,31 +30,27 @@
 import java.util.Map;
 
 import org.junit.Assert;
-import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.ClassRule;
 import org.junit.Rule;
 import org.junit.Test;
-import org.junit.runner.RunWith;
 
-import org.mockito.runners.MockitoJUnitRunner;
-
-import org.powermock.api.mockito.PowerMockito;
+import org.mockito.Mockito;
 
 import org.skyscreamer.jsonassert.JSONAssert;
 
 /**
  * @author Mateus Santana
  */
-@RunWith(MockitoJUnitRunner.class)
```

Comment:
@JanisZhang What's the reason to remove `@RunWith(MockitoJUnitRunner.class)` here, but keep it in `DDMExpressionFactoryImplTest`?

Reply:
@dantewang thanks... I forgot.